### PR TITLE
Added AddTier Modal

### PR DESCRIPTION
**Pull request by plxity**
I have added 'Add Tier Modal'. Please let me know how can it be improved and what changes should be done?

### CODE REVIEW

File: components/AddTierModal/index.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+const ModalContainer = styled.div`
+.absolute;
+.top-50%;
+.left-50%;
+transform: translate(-50%,0%);
+background: white;
+.rounded;
+width:40vw;
+height:400px;
```
Comment (vilvaathibanpb) [inline components/AddTierModal/index.js:11]: with fixed height, dont you think, we need an `overflow-y: scroll`?

File: components/AddTierModal/index.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
+import styled from 'styled-components';
+
+const ModalContainer = styled.div`
+.absolute;
```
Comment (vilvaathibanpb) [inline components/AddTierModal/index.js]: Shouldnt this be `fixed`?

**Review by vilvaathibanpb — COMMENTED** [review]
@plxity Great job
Just a small observation. But overall a great work 🥇 . I wasn't able to run it as well as you haven't push upstream I guess. However after you resolve my comments, I will merge and test it in master

File: components/AddTierModal/index.js (hunk 1 above)
Comment (plxity) [reply components/AddTierModal/index.js:11]: Yes, I will add that property.

**Review by vilvaathibanpb — APPROVED** [review]

**Pull request merged**